openforcefield/topology/molecule.py

```diff
@@ -4680,7 +4680,7 @@ def get_structure_string(self):
         memfile.seek(0)
         block = memfile.getvalue()
         # FIXME: Prevent multi-model PDB export with a keyword in molecule.to_file()?
-        models = block.split('END\n')
+        models = block.split('END\n', 1)
```

Comment:
I'd like some discussion on this workaround. 

(I think) `nglview` accepts multimodel PDBs, but I think it only considers the first one by default. Do we want to pass only the first one to begin with? Is  `END` the right keyword here? Or is it `ENDMDL`? OpenEye generates multiple conformers with `END` tags to separate them.

Reply:
I don't have a recommendation here but you seem to be right about what nglview [does](https://github.com/arose/nglview/issues/765#issuecomment-459862450) and OpenEye does write it with `END`, not `ENDMDL`

Can RDKit not write it, or am I calling something wrong here?
```
In [21]: mol = Molecule.from_smiles('CCOCOCOCCOCCOCOCC')                                       

In [22]: mol.generate_conformers()                                                             

In [23]: mol.to_file('rdkit.pdb', file_format='pdb', toolkit_registry=RDKitToolkitWrapper())   
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-23-6634d46173bd> in <module>
----> 1 mol.to_file('rdkit.pdb', file_format='pdb', toolkit_registry=RDKitToolkitWrapper())

~/software/openforcefield/openforcefield/topology/molecule.py in to_file(self, file_path, file_format, toolkit_registry)
   3577         if type(file_path) == str:
   3578             # Open file for writing
-> 3579             toolkit.to_file(self, file_path, file_format)
   3580         else:
   3581             toolkit.to_file_obj(self, file_path, file_format)

~/software/openforcefield/openforcefield/utils/toolkits.py in to_file(self, molecule, file_path, file_format)
   2006         # open a file object and pass to the object writer
   2007         with open(file_path, 'w') as file_obj:
-> 2008             self.to_file_obj(molecule=molecule, file_obj=file_obj, file_format=file_format)
   2009 
   2010     def canonical_order_atoms(self, molecule):

~/software/openforcefield/openforcefield/utils/toolkits.py in to_file_obj(self, molecule, file_obj, file_format)
   1979         try:
   1980             writer = self._toolkit_file_write_formats[file_format](file_obj)
-> 1981             writer.write(rdmol)
   1982             writer.close()
   1983         # if we can not write to that file type catch the error here

ValueError: Bad Conformer Id

Reply:
Yep, we have a bug there, as I commented in https://github.com/openforcefield/openforcefield/pull/560#issuecomment-608375591. I didn't have the time to double check  this can happen in other parts of the library (99% yes), so I guess we should raise an issue for this bug.